packages/pipeline-services/src/migration/index.ts

```diff
@@ -21,19 +21,26 @@ import migrateV2 from "./migrateV2";
 import migrateV3 from "./migrateV3";
 import migrateV4 from "./migrateV4";
 
-export function migrate(pipelineJSON: any) {
+export function migrate(
+  pipelineJSON: any,
+  setNodePathsRelativeToPipeline?: (pipeline: any) => any
+) {
   return produce(pipelineJSON, (draft: any) => {
     const version = draft.pipelines[0].app_data?.version ?? 0;
     if (version < 1) {
+      console.debug("migrating pipeline from v0 to v1");
       migrateV1(draft);
     }
     if (version < 2) {
```

Comment:
Is there a reason we're using ifs instead of else ifs?

Reply:
because they're sequential, so if you enter one you would enter all of the following ones, but you don't always enter all of them.